_posts/2021-03-20-sr2021-third-league-sessions.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: Great Performance from teams during the SR2021 Third League Session
+---
+
+With another 36 matches of the SR2021 Competition, team HRS4 from Hills Road
+Sixth Form College head up the league, though a number of other
+teams are hot on their heels. This weekend's matches saw great performances from
+our teams, including a new team, SWI from South Wilts Grammar School, who hadn't submitted a robot for the first two league sessions.
+
+Module Ⅲ provided a new challenge to the teams, both with the change to end state scoring and by the addition of a lockout mechanic for territories. The latter in particular created
+some tense moments during the matches, with robots able to dramatically affect
+each others' scores by booby trapping territories.
+
+At the end of the league we announced the details of our
+[fourth and final game module](https://studentrobotics.org/docs/resources/2021/rulebook.html).
+This substantially changes the arena layout and adds new kinds of territories worth more game points. The towers also received an upgrade meaning they no longer become locked out, but if a robot attempts to claim a territory they already own then it will become unclaimed.
+
+This module of the game requires that the teams robots further increase their
+awareness of their environment and how they react to it if they want to claim
+territories and also hold them for the duration of the match
+
+We're now three quarters of the way through the league matches that will seed the eventual
+knockout stage, so teams still have everything to play for. The next league
+session will be in 5 weeks, on the 24th April.
```

Comment:
Maybe add something here about this next league being the last one before the knockouts? Not sure how best to work that in though.

Reply:
9baf159